docs/theme_architecture/blocks/schema/variables/variable_types/experience_slot.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Experience Slot
+parent: Variable types
+has_children: false
+---
+
+# Experience Slot
+
+Renders two or three select inputs. First, the creator can select an experience. Then they can choose a slot date for the chosen experience. Then, if the experience has start times, they can choose a time for the chosen date.
```

Comment:
```suggestion
Renders two or three select inputs. Initially, the creator picks an experience, followed by selecting a slot date for the chosen experience. If the experience has specific start times, they can then choose a time for the selected date.
```

[Nit] Wondering if this is a bit more readable in terms of flow?

Reply:
Ahh yes, thank you, I was following the structure used on [`experience_date`](https://github.com/easolhq/easolhq.github.io/blob/12adbc0afc98cdd3bf98a088e552b816dd72b876/docs/theme_architecture/blocks/schema/variables/variable_types/experience_date.md#L10) - I've reworded here using some of your suggestions, nice narrative flow now 🙌